### DISCUSSION TITLE
UV-K5(8) USB-C port

### DISCUSSION THREAD

**Discussion by rytisp** [egzumer/uv-k5-firmware-custom#498] [General]
what's the purpose of that USB-C port, my radio doesn't charge through it, also can't detect any data traffic on that port.

**Comment by f4axk** [comment]
USB-C does not take data
USB-C does not charge above 80% for safety reasons, you must be
less than 80% to charge.

**Reply by egzumer** [reply]
There is no safety reason. Boards v1.4 does not usually charge to 100% but that is bad design. Boards v1.6 have different charging chip and charge to 100%.

**Reply by f4axk** [reply]
Thank you egzumer for this clarification

I have a PCB version 1.5, in USB-C the charge does not exceed 80%.
On my PCB version 1.6, no problem loading to 100%.

73s

**Reply by BruMcD** [reply]
Does the Serial number give any clue to the PCB version ?

Answering my own questions.  Just checked two that came with 2024 serial numbers, identical until the last 3 digits, only 100 apart, and from the same supplier, same batch and with same 2.01.33 (v. recent factory Firmware).  
1 charges to 80%  the other to 100%
